src/components/ComponentsPage/ComponentsPage.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { EmptyStateBody, PageSection, PageSectionVariants, Title } from '@patternfly/react-core';
+import { FeatureFlagIndicator } from '~/feature-flags/FeatureFlagIndicator';
+import { IfFeature } from '~/feature-flags/hooks';
+import { ExternalLink } from '~/shared';
+import AppEmptyState from '~/shared/components/empty-state/AppEmptyState';
+import emptyStateImgUrl from '../../assets/Components.svg';
+import { LEARN_MORE_ABOUT_COMPONENTS } from '../../consts/documentation';
+import PageLayout from '../PageLayout/PageLayout';
+
+const ComponentsPage: React.FC = () => {
+  return (
+    <IfFeature flag="components-page">
+      <PageLayout
+        title={
+          <>
```

Comment:
super nit: no need to have the `<>...</>` fragment, you can render `<Title ...>` directly :)